eventsourced-projection/Cargo.toml

```diff
@@ -24,6 +24,7 @@ tracing       = { workspace = true }
 trait-variant = { workspace = true }
 
 [dev-dependencies]
+eventsourced           = { path = "../eventsourced", version = "0.22.0", features = [ "test" ] }
```

Comment:
So, this is the testing feature,  you talked about. Are `dev-dependencies` and public features merged automatically (i.e. here `test` and `serde_json` )?